src/style.css

```
@@ -0,0 +1,45 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  margin-right: 4px;
+  margin-top: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  margin: 0 4px;
+}
+
+.star:nth-child(n) {
+  display: flex;
+  flex-direction: row;
+}
```

U selected each child :)

```suggestion
.star {
  display: flex;
  flex-direction: row;
}
```